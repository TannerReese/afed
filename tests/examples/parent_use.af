# Test importing of other files
use "subdir/child_use2.af",
from_child1 = ``,
use "child_use1.af",
from_child2 = ``

