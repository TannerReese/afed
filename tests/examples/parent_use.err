Parse Error in line 4, column 21 of "parent_use.af"
|   use "child_use1.af",
|                       ^
Error: Redefinition of label 'redef' in map

1 Parse Error encountered


