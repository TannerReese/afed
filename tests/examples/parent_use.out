# Test importing of other files
use "subdir/child_use2.af",
from_child1 = `4`,
use "child_use1.af",
from_child2 = `5`

